packaging/overview.md

```diff
@@ -0,0 +1,50 @@
+# Code Packaging
+
+Learning goals:
+
+- Understanding the necessity to somehow package code written in a crude way
+- Being able to determine the type of packaging required for the code
+- Modifying code and packaging it according to a standardization
+- Uploading code to a remote package managing platform and testing the installation
+
+## Survey
+
+| Duration | Format | Material |
+| --- | --- | --- |
+| 15 minutes | poll | [packaging_survey.md](https://github.com/Simulation-Software-Engineering/Lecture-Material) |
```

Comment:
- Do you think this will take 15 minutes?
- You could update the link already so you do not forget it later

```suggestion
| 15 minutes | poll | [packaging_survey.md](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/packaging/overview.md) |
```